tests/gabc-dump/glyphs/protrusion.exp

```diff
@@ -14,13 +14,15 @@ et(i) (::)
 et(i) (::Z)
 Nus:(i) *(,)
 et(i) (::)
-Dus<pr>:(i) *(,)
+Nus$:(i) *(,)
```

Comment:
What's going on here?  The changed lines seem to be mismatched.

Reply:
I added a case showing an escaped `:` as not having protrusion meaning.

Reply:
So it's just that the added case is similar enough to an existing line that the diff tool gets confused when trying to match up the lines?

Reply:
Yes.